Cargo.toml

```diff
@@ -18,5 +18,6 @@ phf = { version = "0.7", features = ["unicase"] }
 unicase = "1.1"
 
 [build-dependencies]
+phf = { version = "0.7", features = ["unicase"] }
```

Comment:
Can you add a comment explaining why this dep is here so it doesn't get deleted by accident?

Reply:
Done.